src/openloco/windows/industry_list.cpp

```diff
@@ -640,7 +640,7 @@ namespace openloco::ui::windows::industry_list
             if (self->var_846 == 0xFFFF)
                 industryCost = dword_E0C39C;
 
-            if ((self->var_846 == 0xFFFF && dword_E0C39C == (1 << 31)) || self->var_846 != 0xFFFF)
+            if ((self->var_846 == 0xFFFF && dword_E0C39C == uint32_t(1 << 31)) || self->var_846 != 0xFFFF)
```

Comment:
For future reference:
```suggestion
            if ((self->var_846 == 0xFFFF && dword_E0C39C == (1ULL << 31)) || self->var_846 != 0xFFFF)
```

Reply:
Cool, thanks!